_unittest/conftest.py

```diff
@@ -70,7 +70,7 @@
         default_version = sys.modules["__main__"].oDesktop.GetVersion()[0:6]
     config = {
         "desktopVersion": default_version,
-        "NonGraphical": True,
+        "NonGraphical": False,
```

Comment:
```suggestion
        "NonGraphical": True,
```

Reply:
Thanks @maxcapodi78 . It was modified for debugging purpose as I was working with the modeler and wanted to make sure that the spline were properly drawn.